src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -68,19 +68,21 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
 
             foreach (string sourceAssembly in sources)
             {
+                var sourceAssemblyPath = Path.IsPathRooted(sourceAssembly) ? sourceAssembly : Path.Combine(Directory.GetCurrentDirectory(), sourceAssembly);
```

Comment:
This will work so long as we are sure that any relative path we get is relative to the current directory at the time of running the test. We never change it, and hopefully VS doesn't do that either.